src/main/scala/edu/berkeley/ce/rockslicing/BlockVTK.scala

```diff
@@ -36,18 +36,18 @@ object BlockVTK {
     // vertical line running through this center.
     // Check if points are on opposite sides of the center. Points left of center will be before points
     // right of the center
-    if ((pointA(0) - center(0) < -NumericUtils.EPSILON) && (pointB(0) - center(0) >= NumericUtils.EPSILON)) {
```

Comment:
Based on our discussion, I'm going to change it back to `EPSILON` and add an issue about these small blocks and ordering their vertices.